README.md

```diff
@@ -1,3 +1,20 @@
-# cascade-layers-polyfill
+# cascade-layers-polyfill-demo
 
-Run `npx webpack` to build the project after clonining. Navigate to the `index.html` of the project in your browser of choice to view.
+This is a demo showing how the Cascade Layers Polyfill is used.
+
+
+## Local Testing
+
+To run this project locally, run `npx webpack` to build the project.
+Open `src/index.html` in your web browser.
+
```

Comment:
@mirisuzanne @sanajaved7 @oluoluoxenfree I think we still need webpack if someone were to clone this to work on it locally?

Reply:
I think webpack is needed for the polyfill to work on this demo, as we're using a node module for the post css plugin.Will put it back, sorry for the confusion